packages/babel-types/src/definitions/experimental.js

```diff
@@ -10,6 +10,15 @@ import {
   classMethodOrDeclareMethodCommon,
 } from "./es2015";
 
+defineType("ArgumentPlaceholder", {
+  visitor: ["argument"],
+  fields: {
+    argument: {
+      validate: assertNodeType("ArgumentPlaceholder"),
```

Comment:
This would mean that the AST is something like this:
```jsonc
{
  "type": "ArgumentPlaceholder",
  "argument": {
    "type": "ArgumentPlaceholder",
    // ...
  }
}
```

It should only be this:
```js
defineType("ArgumentPlaceholder", {});
```